book/src/setup_codegen.md

```diff
@@ -1,11 +1,3 @@
-- [`flutter_rust_bridge_codegen`](https://lib.rs/crates/flutter_rust_bridge_codegen), the core codegen for Rust-Dart glue code
-- [`ffigen`](https://pub.dev/packages/ffigen), to generate Dart code from C headers
-- [`just`](https://github.com/casey/just), a modern command runner alternative to Make
-- A working installation of LLVM, see [Installing LLVM](https://pub.dev/packages/ffigen#installing-llvm)
+# Setup codegen
```

Comment:
Weird... This file seems to never be used (i.e. not in SUMMARY.md - https://github.com/fzyzcjy/flutter_rust_bridge/blob/master/book/src/SUMMARY.md).

So I guess this whole file can be deleted?

Reply:
removed in 15d6e46fb46c815154c1921a6517f7c47481b8d5